custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt

```diff
@@ -45,7 +39,8 @@ class ScenarioSimilarities(
         browseBoardsPage: BrowseBoardsPage,
         createIssue: Action,
         searchWithJql: Action,
-        browseProjects: Action
+        browseProjects: Action,
+        projectIssueNavigatorAction: ProjectIssueNavigatorAction
```

Comment:
We can leave it open-ended like for others:
```suggestion
        projectIssueNavigatorAction: Action
```